components/post_view/post_list_virtualized/post_list_virtualized.tsx

```diff
@@ -46,91 +45,95 @@ const virtListStyles = {
 
 const OFFSET_TO_SHOW_TOAST = -50;
 
-export default class PostList extends React.PureComponent {
-    static propTypes = {
-
-        /**
-         * Array of Ids in the channel including date separators, new message indicator, more messages loader,
-         * manual load messages trigger and postId in the order of newest to oldest for populating virtual list rows
-         */
-        postListIds: PropTypes.array.isRequired,
-
-        /**
-         * Set to focus this post
-         */
-        focusedPostId: PropTypes.string,
-
-        /**
-         * The current channel id
-         */
-        channelId: PropTypes.string.isRequired,
-
-        /**
-         * Used for disabling auto retry of posts and enabling manual link for loading posts
-         */
-        autoRetryEnable: PropTypes.bool,
-
-        /**
-         * used for populating header, scroll correction and disabling triggering loadOlderPosts
-         */
-        atOldestPost: PropTypes.bool,
-
-        /**
-         * used for disabling triggering loadNewerPosts
-         */
-        atLatestPost: PropTypes.bool,
-
-        /**
-         * used in passing to post row for enabling animation when loading posts
-         */
-        loadingNewerPosts: PropTypes.bool,
-        loadingOlderPosts: PropTypes.bool,
-
-        latestPostTimeStamp: PropTypes.number,
-
-        lastViewedAt: PropTypes.string,
-
-        isMobileView: PropTypes.bool.isRequired,
-
-        actions: PropTypes.shape({
-
-            /**
-             * Function to get older posts in the channel
-             */
-            loadOlderPosts: PropTypes.func.isRequired,
-
-            /**
-             * Function to get newer posts in the channel
-             */
-            loadNewerPosts: PropTypes.func.isRequired,
-
-            /**
-             * Function used for autoLoad of posts incase screen is not filled with posts
-             */
-            canLoadMorePosts: PropTypes.func.isRequired,
-
-            /**
-             * Function to check and set if app is in mobile view
-             */
-            checkAndSetMobileView: PropTypes.func.isRequired,
-
-            /**
-             * Function to change the post selected for postList
-             */
-            changeUnreadChunkTimeStamp: PropTypes.func.isRequired,
-
-            updateNewMessagesAtInChannel: PropTypes.func.isRequired,
-        }).isRequired,
-    }
+type Props = {
+
+    /*
```

Comment:
Can you actually switch these back to being wrapped in `/** **/` comments? Even though those are a bit longer, they're picked up by various tools as JSDoc comments, and they'll be displayed in editors when hovering over those fields.

That only needs to be done for comments in the type definitions though. I see some other spots further down where you switched single-line `/* */` comments to `//`, and that makes sense to me